cmp/grid/filter/GridFilterModel.js

```diff
@@ -22,6 +22,8 @@ export class GridFilterModel extends HoistModel {
     gridModel;
     /** @member {(Store|View)} */
     bind;
+    /** @member {boolean} */
+    autoApply;
```

Comment:
should this be `immediateApply` it does still get `auto`matically applied to the bound store

Reply:
or perhaps, `commitOnChange` ?

Reply:
Will change to `commitOnChange`... I notice that's what we use for the equivalent setting on the Column Chooser